app/views/hosts/_compute_detail.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<% new_host = @host.nil? || new_host?(@host) %>
+
+<%= render :partial => provider_partial(compute_resource, 'base'),
+            :locals => { :f => f, :compute_resource => compute_resource, :new_host => new_host }.merge(args_for_compute_resource_partial(@host)) %>
+
+<!--NICS-->
+<% if provider_partial_exist?(compute_resource, 'network') %>
+  <fieldset>
+    <legend><%= _("Network interfaces") %></legend>
+    <p class="help-block">
+      <%= _("Network interfaces management has been moved to the Interfaces tab.") %>
+      <%= _("Please set your interfaces") %> <a href="#" onclick="$('#network_tab a').click(); return false;"><%= _("there") %></a>.
```

Comment:
i18n - this is going to be very hard for anybody to translate (I realise it was already merged, but it wasn't picked up then).

I'd either come up with an entire phrase that you can make a link, remove the whole thing, or maaaybe use %{link_start} and %{link_end} to insert the pieces of HTML into the translated string (and some .html_safe).